recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -91,32 +91,27 @@ PACKAGECONFIG[thunder] = "-DENABLE_THUNDER=ON,-DENABLE_THUNDER=OFF,virtual/open-
 PACKAGECONFIG[video-plane-display-dmabuf] = "-DUSE_WPE_VIDEO_PLANE_DISPLAY_DMABUF=ON,-DUSE_WPE_VIDEO_PLANE_DISPLAY_DMABUF=OFF,wpebackend-fdo"
 PACKAGECONFIG[webxr] = "-DENABLE_WEBXR=ON,-DENABLE_WEBXR=OFF,openxr"
 PACKAGECONFIG[jit] = "-DENABLE_JIT=ON -DENABLE_C_LOOP=OFF,-DENABLE_JIT=OFF -DENABLE_C_LOOP=ON,"
-PACKAGECONFIG[gold] = "-DUSE_LD_GOLD=ON,-DUSE_LD_GOLD=OFF,"
```

Comment:
If you want to keep the `PACKAGECONFIG` option, what you can do is adding `-fuse-ld=gold` to the linker flags, which of course works in _all_ versions of WebKit, regardless of the `USE_LD_GOLD` option. Actually, I would argue that the `-fuse-ld=` linker flag is the only supported way of changing linkers, because `USE_LD_GOLD` is an internal (not public) WebKit build option (which is why I removed it recently).

Reply:
I think we cand let it go and manage this option in the distro level. There is the `ld-is-gold` flag for DISTRO_FEATURES  in Yocto for this purpose.

Reply:
If it can be managed at the distro level instead of in each package, then even better 👍🏼